basic/exercise5/script5.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 
 function findFirstYearStartingFromSunday(startYear, endYear) {
```

Comment:
You should rename it as it returns all years from specific range meeting the criteria